src/lib/api/Posts.js

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+import {API_DOMAIN} from "./PathList";
+
+
+export async function getListOfPosts(authStr, userId) {
+    return axios.get("http://" + API_DOMAIN + "/api/posts/", { headers: { Authorization: authStr },
+        params: { user_id: userId,}})
+        .then(result => result.data);
+}
+
+export async function getFeed(authStr) {
+    return axios.get("http://" + API_DOMAIN + "/api/posts/", { headers: { Authorization: authStr },
+        params: { feed: true,}})
+        .then(result => result.data);
+}
+
+export function postPost(authStr, credentials) {
+    return axios.post("http://" + API_DOMAIN + "/api/posts/create/", credentials, {
+        headers: {Authorization: authStr}
+    }).catch((error) => {
+        console.log(error.response);
```

Comment:
We probably need to show correct error message to the user. For now, it is not necessary, but it should be updated in the future. So let's add a `todo` here and in other places were we just log the error

Reply:
Ok